test/contrib/test_pyopenssl.py

```diff
@@ -55,3 +57,11 @@ def test_dnsname_to_stdlib_leading_splat(self):
         expected_result = '*.xn--p1b6ci4b4b3a.xn--11b5bs8d'
 
         self.assertEqual(_dnsname_to_stdlib(name), expected_result)
+
+    def test_inject_validate_fail(self):
+        """
+        Injection should not be supported if we are missing required dependencies.
+        """
+        with patch("cryptography.x509.extensions.Extensions") as mock:
+            del mock.get_extension_for_class
+            self.assertRaises(ImportError, inject_into_urllib3)
```

Comment:
So this test is going to be a bit problematic because the module is *already* injected into urllib3 thanks to `setup_module` in this file. You may want to pull this out into a different test file (possibly a brand new one).